packages/metapackage/package.json

```
@@ -21,7 +21,9 @@
   "files": [
     "lib/*.d.ts",
     "lib/*.js.map",
-    "lib/*.js"
+    "lib/*.js",
+    "style/index.css",
+    "style/index.js"
```

```suggestion
    "lib/*.js"
```

Left over that can be cleaned.